tardis/visualization/tools/liv_plot.py

```diff
@@ -192,16 +192,15 @@ def _generate_plot_data(self, packets_mode):
 
         for specie_list in self._species_mapped.values():
             full_v_last = []
-            for specie in specie_list:
-                if specie in self.species:
-                    if specie not in groups.groups:
-                        atomic_number = specie // 100
-                        ion_number = specie % 100
+            for element in specie_list:
```

Comment:
species

Reply:
I felt it would be confusing because in line 196 we would then check if species in self.species